src/extension.ts

```diff
@@ -828,6 +830,8 @@ function launchMetals(
         });
       });
 
+      registerCommand(`metals.symbol-search`, () => openSymbolSearch(client));
```

Comment:
I would still add a comment here referencing the upstream issue, so that we remember why we need this. Something like

```suggestion
      // NOTE: we offer a custom symbol search command to work around the limitations of the built-in one, see https://github.com/microsoft/vscode/issues/98125 for more details.
      registerCommand(`metals.symbol-search`, () => openSymbolSearch(client));
```